src/app/sport/[sport]/page.tsx

```diff
@@ -0,0 +1,60 @@
+import prisma from '@db/prismaSingleton'
+
+import SelectGrid from '@components/SelectGrid'
+import TopBar from '@components/TopBar'
+
+type PageProps = {
+  params: { sport: string },
+}
+
+const Page = async (props: PageProps) => {
+  const {
+    params: { sport: sportKey },
+  } = props
+
+  const sport = await prisma.team.findUnique({
+    where: {
+      key: sportKey,
+    },
+  })
+  const { id: sportId, title: sportTitle } = sport || {}
+
+  const competitions = await prisma.competition.findMany({
```

Comment:
instead of querying for the sport, getting the id and making a 2nd query for the competitions, i would run a single query to get the competitions by sportKey
```
const competitions = await prisma.competition.findMany({
  where: {
    sport: {
      key: sportKey,
    },
  },
})
```

Reply:
I've updated as per your suggestion here for now because we only have 2 sports (Well 1 really) and its a 1 word sport.

So we can use the sportKey to display the 'SELECT A RUGBY COMPETITION'

On the Teams Page I've kept the competition and teams queries because using the competition key would display "SELECT A SUPERRUGBY TEAM" when we want to use the competition title to get "SELECT A SUPER RUGBY TEAM".

Not sure whether its better to have 2 queries or make a function to separate camel cased words or just create a function with the hardcoded titles. For now I think this will do!

Reply:
nice, i was thinking about this last night and i would do the same, i saw you kept the 2 queries for competition